references/obj_detection/train_pytorch.py

```diff
@@ -185,7 +212,8 @@ def main(args):
             })
         if f1_score > max_score:
             print(f"Validation metric increased {max_score:.6} --> {f1_score:.6}: saving state...")
-            torch.save(model.state_dict(), f"./{exp_name}.pt")
+            # torch.save(model.state_dict(), f"./{exp_name}.pt")
+            torch.save(model.state_dict(), f"./{epoch}.pt")
```

Comment:
I'm not sure that will help. I'd argue we should keep the same naming, if you want to save every epoch, you can add a `torch.save` outside of this "if"

Reply:
Thanks, I forgot to change this back to the way it was.